app/views/attendances/_attendance_table.html.erb

```diff
@@ -28,7 +28,7 @@
         <%= attendance.login_time.to_formatted_s(:long_ordinal) %>
       </td>
       <td>
-        <% if attendance.logout_time.blank? %>
+        <% if logout_added?(attendance) %>
```

Comment:
add all of this if condition to the helper function

Reply:
it is in the helper